hathor/cli/generate_keys.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Hathor Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+from cryptography.hazmat.primitives.asymmetric import ec
+from cryptography.hazmat.primitives.asymmetric.ec import EllipticCurvePrivateKeyWithSerialization
+
+
+def main():
+    from hathor.cli.util import create_parser
+    from hathor.crypto.util import (
+        get_address_b58_from_public_key,
+        get_private_key_bytes,
+        get_public_key_bytes_compressed,
+    )
+    parser = create_parser()
+    parser.add_argument('--config-yaml', type=str, help='Configuration yaml filepath')
+    args = parser.parse_args(sys.argv[1:])
+    if not args.config_yaml:
+        raise Exception('`--config-yaml` is required')
+    os.environ['HATHOR_CONFIG_YAML'] = args.config_yaml
+
+    private_key = ec.generate_private_key(ec.SECP256K1())
+    public_key = private_key.public_key()
+    assert isinstance(private_key, EllipticCurvePrivateKeyWithSerialization)
+
+    print('Private key (hex):', get_private_key_bytes(private_key=private_key).hex())
```

Comment:
Maybe we should have an `output_file` argument so we would write to it directly (or to stdin by default).

Reply:
But would we have a parser to read the contents of the file in a useful way? I guess we should generate one or more files that are ready to be used.

Reply:
I changed it to output a json string like `gen_peer_id` in https://github.com/HathorNetwork/hathor-core/pull/1041/commits/7ba3d0fe840d424e4f83c6d2434ff228108d405c. I still have to change the CLI to read this output as a file in the next PR, so you can leave this thread open for now.

Reply:
Great! Thanks!